docs/source/contrib_docs/doc-new-build.md

```diff
@@ -0,0 +1,33 @@
+# Building automatically on ReadTheDocs
```

Comment:
This seems like a general description of how to use RTD. I don't think that belongs in our docs (beyond saying 'We use RTD' and linking to *their* docs). We've learned before that general info in our own developer docs rapidly ends up out of date and misleading (last time it was info on how to use git).